build/packaging/linux/bin/kcnd

```diff
@@ -152,22 +152,41 @@ start() {
         OPTIONS="$OPTIONS --verbosity $VERBOSITY"
     fi
 
+    #Deprecated option
     if [ ! -z $TXPOOL_EXEC_SLOTS_ALL ]; then
```

Comment:
```suggestion
    if [ ! -z $TXPOOL_EXEC_SLOTS_ALL ]; then # for v1.6.4 and after, use txpool.globalslots instead
```

Reply:
```suggestion
    #Deprecated option
    if [[ ! -z $TXPOOL_EXEC_SLOTS_ALL ]] || [[ ! -z $TXPOOL_GLOBAL_SLOTS ]]; then
```
Or we could combine the two checkings into one?

Reply:
Thanks for good review winnie!
I reflected it in the first way you suggested.
Second, I thought it would be too much work to erase it in the future.